src/usermgmt.rs

```diff
@@ -27,23 +30,32 @@ fn create_kdf(pepper: &[u8]) -> Argon2 {
     Argon2::new_with_secret(pepper, Argon2id, V0x13, params).expect("failed to initialize Argon2")
 }
 
-struct Session(String);
+#[derive(Serialize, Debug, sqlx::FromRow)]
+#[serde(rename_all = "camelCase")]
+pub struct AccountSession {
+    pub id: i64,
+    email: String,
+    #[serde(skip_serializing)]
+    session_id: Vec<u8>,
+}
 
-impl Session {
-    async fn create(uid: i64, db: &DB) -> eyre::Result<Self> {
+impl AccountSession {
```

Comment:
Are we going to have *non-account* sessions?

Reply:
I don't have such plans, but this ties into not being fully happy with `AccountSession`. The "Account" there is not "as opposed to non-accounts" but "as opposed to a type that represents _only_ an `Account` or _only_ a `Session`".

If the sql embedded directly in the handlers around `Account` is extracted and turned into a real type, this will probably just be split into two types. Happy to hear suggestions on a name in the mean time though.